guacamole/src/main/frontend/src/translations/en.json

```diff
@@ -183,6 +183,79 @@
+
+        "HELP_JSON_HEADER": "JSON Format",
+        "HELP_JSON_DESCRIPTION": "A connection import JSON file is a list of connection objects. At minimum the connection name and protocol must be specified in each connection object.",
+        "HELP_JSON_MORE_DETAILS": "The connection group ID that the connection should be imported into may be directly specified with a \"parentIdentifier\" field, or the path to the parent group may be specified using a \"group\" field as shown below. An array of user and user group identifiers to grant access to may be specified per connection.",
+
+        "HELP_YAML_HEADER": "YAML Format",
+        "HELP_YAML_DESCRIPTION": "A connection import YAML file is a list of connection objects with exactly the same structure as the JSON format.",
+        
+        "HELP_SEMICOLON_FOOTNOTE": "If present, semicolons can be escaped with a backslash, e.g. \"first\\\\;last\"",
+
+        "ERROR_AMBIGUOUS_CSV_HEADER": 
+            "Ambiguous CSV Header \"{HEADER}\" could be either a connection attribute or parameter",
+        "ERROR_ARRAY_REQUIRED": 
+            "The provided file must contain a list of connections",
+        "ERROR_DUPLICATE_CSV_HEADER": 
+            "Duplicate CSV Header: {HEADER}",
+        "ERROR_EMPTY_FILE": "The provided file is empty",
+        "ERROR_INVALID_CSV_HEADER": 
+            "Invalid CSV Header \"{HEADER}\" is neither an attribute or parameter",
+        "ERROR_INVALID_GROUP": "No group matching \"{GROUP}\" found",
+        "ERROR_INVALID_FILE_TYPE": 
+            "Unsupported file type: \"{TYPE}\"",
+        "ERROR_INVALID_USER_IDENTIFIERS":
+            "Users not found: {IDENTIFIER_LIST}",
+        "ERROR_INVALID_USER_GROUP_IDENTIFIERS":
+            "User Groups not found: {IDENTIFIER_LIST}",
+        "ERROR_NO_FILE_SUPPLIED": "Please select a file to import",
+        "ERROR_AMBIGUOUS_PARENT_GROUP":
+            "Both group and parentIdentifier may be not specified at the same time",
+        "ERROR_REQUIRED_PROTOCOL": 
+            "No connection protocol found in the provided file",
+        "ERROR_REQUIRED_NAME": 
+            "No connection name found in the provided file",
+
+        "ERROR_FILE_SINGLE_ONLY": "Please upload only a single file at a time",
+
+        "TABLE_HEADER_NAME" : "Name",
+        "TABLE_HEADER_PROTOCOL" : "Protocol",
+        "TABLE_HEADER_ERRORS" : "Errors",
+        "TABLE_HEADER_ROW_NUMBER": "Row #",
+
+        "UPLOAD_FILE_TYPES": "CSV, JSON, or YAML",
+        "UPLOAD_HELP_LINK": "View Format Tips",
+        "UPLOAD_DROP_TITLE": "Drop a File Here",
+        "UPLOAD_BROWSE_LINK": "Browse for File"
+        
+    },
```

Comment:
Seems like the style, here, doesn't match the other sections, in a couple of ways:
* Generally things are organized alphabetically, and so `ERROR_` should be between `DIALOG_` and `FIELD_`.
* There are several line breaks here between the key and value pairs.